src/r_main.c

```diff
@@ -295,7 +295,7 @@ static void R_InitTextureMapping (void)
         ;
       xtoviewangle[x] = (i<<ANGLETOFINESHIFT)-ANG90;
       // [FG] linear horizontal sky scrolling
-      linearskyangle[x] = ((viewwidth/2-x)*((video.unscaledw<<6)/viewwidth))*(ANG90/(NONWIDEWIDTH<<6));
+      linearskyangle[x] = ((viewwidth/2-x)*((video.unscaledw<<FRACBITS)/viewwidth))*(ANG90/(NONWIDEWIDTH<<FRACBITS));
```

Comment:
Why are `<<6` used here? It seems that `<<FRACBITS` decrease "jumps" when the resolution changes.

Reply:
Frankly, I don't have a clue. 🤷

Reply:
> Why is `<<6` used here? It seems that `<<<FRACBITS` reduces the "jump" when the resolution changes.

I don't remember why either other than it was the result of refactoring https://github.com/ZDoom/gzdoom/commit/1c2dcad for my initial patch for Crispy Doom. Possibly it was done with Crispy's fixed resolution in mind.